src/test/java/reposense/git/GitLogTest.java

```diff
@@ -77,6 +77,47 @@ public void gitLog_authorIgnoreAllJavaFiles_success() {
         Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(2, content));
     }
 
+    @Test
+    public void gitLog_authorIgnoreValidFilePath_success() {
+        config.setFormats(Collections.singletonList(new Format("java")));
+        Author ignoreValidFileAuthor = getAlphaAllAliasAuthor();
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("**collate"));
+        String content = GitLog.get(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(8, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("collate*"));
+        content = GitLog.get(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(8, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("collate.md"));
+        content = GitLog.get(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(8, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("./java"));
+        content = GitLog.get(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(8, content));
```

Comment:
Doesn't seems quite right to have all the expected result to be the same when its working.
In order word, this doesn't seems to be the best way to test it.

Could you create a new testutil to sum and compared the number of files changed instead?